src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/CircularShellRenderer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using ElmSharp;
+using System;
+using System.Linq;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+using XShell = Xamarin.Forms.Shell;
+
+[assembly: ExportRenderer(typeof(XShell), typeof(Tizen.Wearable.CircularUI.Forms.Renderer.CircularShellRenderer))]
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class CircularShellRenderer : Xamarin.Forms.Platform.Tizen.Watch.ShellRenderer
+    {
+
+        NavigationDrawer _drawer;
+        NavigationView _navigationView;
+
+        public CircularShellRenderer()
+        {
+            RegisterPropertyHandler(CircularShell.FlyoutIconBackgroundColorProperty, UpdateFlyoutIconBackgroundColor);
+            RegisterPropertyHandler(CircularShell.FlyoutForegroundColorProperty, UpdateFlyoutForegroundColor);
+        }
+
+        protected override NavigationDrawer CreateNavigationDrawer(EvasObject parent)
```

Comment:
Should use alias as below to avoid build error.
```cs
using NavigationDrawer = Xamarin.Forms.Platform.Tizen.Watch.NavigationDrawer;
using NavigationView = Xamarin.Forms.Platform.Tizen.Watch.NavigationView;
```